packages/react/documentation/coding-conventions.md

```diff
@@ -0,0 +1,31 @@
+# Coding Conventions
+
+## Barrel files
+
+In our React package we use barrel files (`index.ts` in the root and in every component folder) to simplify the import of React components.
+Barrel files allow a consumer to do this:
+
+`import { Heading, Link, Paragraph } from '@amsterdam/design-system-react'`
+
+Instead of this:
+
+```js
+import Heading from "@amsterdam/design-system-react/dist/Heading/Heading";
+import Link from "@amsterdam/design-system-react/dist/Link/Link";
+import Paragraph from "@amsterdam/design-system-react/dist/Paragraph/Paragraph";
```

Comment:
```suggestion
import Heading from "@amsterdam/design-system-react/dist/heading/Heading";
import Link from "@amsterdam/design-system-react/dist/link/Link";
import Paragraph from "@amsterdam/design-system-react/dist/paragraph/Paragraph";
```

I think directory names should be lower snake case and will submit a PR for this asap. We could already apply this here.

Reply:
I think having directory names for React components starting with a capital is a more common convention than snake case, any reason to change this?

Reply:
I wasn't aware of the convention and I understand it, especially for two-word components which would have different spelling in their directory and file name, so let’s follow along.